docs/archive/doc/markdown/use-bundle-installer.md

```diff
@@ -121,6 +121,7 @@ vrli_vrops_server_port
 vrli_vrops_server_user
 vrli_vrops_server_password
 vrli_vrops_server_auth_source
+vrli_package_import_overwrite_mode
```

Comment:
Don't modify the old documentation. Lets add this to the new one?

Reply:
Updating the installer docs is stated in the common task-list for the PR